packages/terra-core-docs/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * Updated
   * Updated Draggable list item example for selection state in `terra-list`.
+  * Updated `terra-html-table` documentation with deprecation notice and badge.
```

Comment:
I have added a CHANGELOG file entry per that change, now PR can be merged

Reply:
Thanks @adoroshk - I will follow the process you describe